examples/02-plot/mesh-picking.py

```diff
@@ -35,7 +35,7 @@
 def callback(mesh):
     """Shrink the mesh each time it's clicked."""
     shrunk = mesh.shrink(0.9)
-    mesh.overwrite(shrunk)  # must operate "in-place" by overwrite
+    mesh.copy_from(shrunk)  # must operate "in-place" by overwrite
```

Comment:
The comment sounded a bit clumsy to begin with, but now it's even more off. How about something like
```suggestion
    mesh.copy_from(shrunk)  # make operation "in-place" by replacing the original mesh
```

Reply:
I like this version, thanks